cmd/kubeadm/app/util/system/package_validator.go

```diff
@@ -127,8 +126,7 @@ func (validator *packageValidator) validate(packageSpecs []PackageSpec, manager
 		// Get the version of the package on the running machine.
 		version, err := manager.getPackageVersion(packageName)
 		if err != nil {
-			klog.V(1).Infof("Failed to get the version for the package %q: %s\n", packageName, err)
-			errs = append(errs, err)
+			errs = append(errs, errors.Wrapf(err, "Failed to get the version for the package %q", packageName))
```

Comment:
I could have just deleted the log as we where already returning the error. but I thought out test made it more clear, what do you think?